src/shared/services/api/api.test.js

```diff
@@ -1,6 +1,5 @@
-import { call, select } from 'redux-saga/effects';
+import { call } from 'redux-saga/effects';
 import request from 'utils/request';
-import { makeSelectAccessToken } from 'containers/App/selectors';
 
 import { generateParams, authCall, authCallWithPayload, authPostCall, authPatchCall } from './api';
 
```

Comment:
the jest.mock('containers/App/selectors' can be removed

Reply:
You're right; I've removed it.